src/main/java/racingcar/global/exception/ExceptionControllerAdvice.java

```diff
@@ -0,0 +1,18 @@
+package racingcar.global.exception;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class ExceptionControllerAdvice {
+
+    private static final String MESSAGE = "입력이 올바르지 않습니다.";
+
+    @ExceptionHandler(MethodArgumentNotValidException.class)
+    public ResponseEntity<String> validExceptionHandle() {
+        return ResponseEntity.badRequest().body(MESSAGE);
+    }
```

Comment:
아래와 같은 방식으로 `@Valid` 에서 발생한 에러의 메시지를 받아올 수 있습니다 :)

```suggestion
    @ExceptionHandler(MethodArgumentNotValidException.class)
    public ResponseEntity<String> validExceptionHandle(MethodArgumentNotValidException e) {
        return ResponseEntity.badRequest().body(e.getAllErrors().get(0).getDefaultMessage());
    }
```

아래 아티클을 통해 힌트를 얻어볼 수 있겠네요!
https://dkswnkk.tistory.com/541